### PR TITLE
Add version numbers

### PR DESCRIPTION
**Pull request by merriam**
Add version numbers to fix bug #71.   Just documentation.

### CODE REVIEW

File: docs/overview.rst [hunk 1]
```diff
@@ -89,10 +90,20 @@ You will get a runtime exception if you try to use it with an older cairo.
...
-Use :func:`cairo_version` to test the version number::
+Use :func:`cairo_version` to test the version number for cairo::
 
     if cairo.cairo_version() > 11000:
         surface.set_mime_data('image/jpeg', jpeg_bytes)
+        
+Here are all the version numbers::
+
+      >>> c = str(cairo.cairo_version())
+      >>> print "The cairo version is", cairo.cairo_version(), " meaning ", c[0] + "." + c[1:3] + "." + c[3:]
```
Comment (SimonSapin) [inline docs/overview.rst]: Please use `cairo_version_string()` here instead of string slicing and concatenation.

File: docs/overview.rst [hunk 2]
```diff
@@ -89,10 +90,20 @@ You will get a runtime exception if you try to use it with an older cairo.
...
+      The cairo version is 11402  meaning  1.14.02
+      >>> print "The pycairo verson is", cairo.version
+      The pycairo verson is 1.10.0
```
Comment (SimonSapin) [inline docs/overview.rst]: Please explain a bit more here: “The latest pycairo version this cairocffi version is compatible with is”

**Comment by merriam** [comment]
Well, take it as it for now.    I truly don't know what “The latest pycairo version this cairocffi version is compatible with is..."; that text pre-existed.   In a future patch, I'll add a function ```cairo.versions()``` that just returns ```{'cairo': { 'string': '1.14.02', 'number': 11402, 'major': 1, 'minor': 14, 'point': 2|, {'pycairo'...}, {'cairocffi':...}```.   If we know compatibilites we can add it there.

I guess it comes down to philosophy:  do you want patches only from people committed to the project or do you want the drive by 'this was an issue so I fixed it' patches?

**Comment by SimonSapin** [comment]
Thanks for your PR. I’ve merged it and made the changes described above in 2e7200a64d9deaafa524463b4ba95d302273e227. The “latest pycairo version” thing is an explanation to go in the docs: pycairo is not actually involved in this code.

I don’t think a `versions()` function would be that useful. In fact I’m considering dropping the pycairo version number altogether, it doesn’t make that much sense there.

**Pull request merged**